shell/src/main/java/alluxio/cli/ConfigurationDocGenerator.java

```diff
@@ -153,10 +153,10 @@ static void writeYMLFile(Collection<? extends PropertyKey> defaultKeys, String f
 
       for (PropertyKey iteratorPK : dfkeys) {
         String pKey = iteratorPK.toString();
-        String description = iteratorPK.getDescription();
+        String description = iteratorPK.getDescription().replace("'", "''");
```

Comment:
This is to escape `'` for single quoted strings in YAML files. YAML files supports multiple ways to declare a string literal, and single quoted string has the least requirement in format or escaped characters. The original issue was that we write description in unquoted strings where jekyll complained about `:` in them.

You can check out the page [here](http://symfony.com/doc/current/components/yaml/yaml_format.html#strings) for a comparison of different string representations.